src/core/server/http/http_service.test.ts

```diff
@@ -76,6 +77,42 @@ test('creates and sets up http server', async () => {
   expect(httpServer.start).toHaveBeenCalledTimes(1);
 });
 
+test('creates and sets up second http server', async () => {
```

Comment:
I think would be nice to write a sort of  integration test to make sure the secondary server is run. one of the examples